tests/rbr/AdjustRegionsTest.java

```diff
@@ -0,0 +1,59 @@
+package rbr;
+
+import java.util.*;
+import org.junit.*;
+import util.*;
+
+public class AdjustRegionsTest {
+
+    private Set<Character> ip = new HashSet<>();
+    private Set<Character> op = new HashSet<>();
+    private Set<Vertex> dst = new HashSet<>();
+    private Region region;
+
+    @Before
+    public void setUp() {
+        ip.add('I');
+        op.add('N');
+        dst.add(new Vertex("0.1"));
+        dst.add(new Vertex("0.2"));
+        dst.add(new Vertex("0.3"));
+        dst.add(new Vertex("1.2"));
+        dst.add(new Vertex("1.3"));
+        dst.add(new Vertex("2.2"));
+        dst.add(new Vertex("2.3"));
+        dst.add(new Vertex("3.3"));
+        region = new Region(ip, dst, op);
+    }
+
+    @Test
+    public void checkIP() throws Exception {
```

Comment:
Name your tests after what you are trying to prove.
I'd name this one 'adjustedRegionsHaveSameInputPortsAsOriginal'.